src/Components/animation/Animation.js

```diff
@@ -42,7 +46,11 @@ class Animation extends React.Component {
    * y2: corresponding vertical position in pixel (=0 if centered)
    */
   _positionPercentToPixel = (x, y) => {
-    return [this.animationWidth * x + this.dLeft, this.animationHeight * y + this.dTop];
+    if (Platform.OS === 'ios') {
```

Comment:
It looks like this is the same formula in either cases, since dleft  equals 0 on ios?

Maybe we could remove the if  to simplify this?